tests/loggers/test_wandb.py

```diff
@@ -182,7 +229,7 @@ def wrapper_something():
     assert params["wrapper_something_wo_name"] == "<lambda>"
 
 
-@mock.patch('pytorch_lightning.loggers.wandb.wandb')
+@ mock.patch('pytorch_lightning.loggers.wandb.wandb')
```

Comment:
accident?

Reply:
Thanks for noticing!
I guess it didn't raise an error because no mock is needed here. I'll update.